### PR TITLE
Add a module to each pipe

### PR DESCRIPTION
**Pull request by squelix**
- Add a module by pipe to be more flexible when importing a pipe.
- Add [Prettier](https://prettier.io/)
- Add `.editorconfig`
- Modify the doc

### CODE REVIEW

**Comment by squelix** [comment]
Up ? 😝
I would like to use this in our project.

File: docs/string.md [hunk 1]
```diff
@@ -353,13 +382,13 @@ Remove accents/diacritics from a string
 ##### File
 
 ```typescript
-import { latinize } from 'angular-pipes';
+import { NgLatinize } from 'angular-pipes';
```
Comment (fknop) [inline docs/string.md]: This module doesn't have Module in its name

File: package.json [hunk 2]
```diff
@@ -1,13 +1,15 @@
 {
   "name": "angular-pipes",
-  "version": "9.1.0",
+  "version": "9.2.0",
```
Comment (fknop) [inline package.json]: Would probably be better to change this to 10.0.0 because of the size of the changes.

File: src/string/with.spec.ts [hunk 3]
```diff
@@ -1,42 +0,0 @@
...
-
-    it('Must return boolean true for start & ends #1', () => expect(pipe.transform(text,'The','Reverse')).toEqual(true));
-    it('Must return boolean true for start & ends #2', () => expect(pipe.transform(text,'the','reverse')).toEqual(true));
-    it('Must return boolean true for start & ends #3', () => expect(pipe.transform(text,'The','Reverse',true)).toEqual(true));
-
-    it('Must return boolean false for start & ends #4', () => expect(pipe.transform(text,'the','reverse',true)).toEqual(false));
-    it('Must return boolean false for start & ends #5', () => expect(pipe.transform(text,'Then','Reverbe',true)).toEqual(false));
-    it('Must return boolean false for start & ends #6', () => expect(pipe.transform(text,'Then','Reverbe')).toEqual(false));
-
-});
```
Comment (fknop) [inline src/string/with.spec.ts]: Test was removed

**Pull request merged**